pkg/controller.v2/controller_tfjob.go

```diff
@@ -114,3 +115,33 @@ func (tc *TFJobController) deletePodsAndServices(tfJob *tfv1alpha2.TFJob, pods [
 	}
 	return nil
 }
+
+func (tc *TFJobController) cleanupTFJob(tfJob *tfv1alpha2.TFJob) error {
+	currentTime := time.Now()
+	ttl := tfJob.Spec.TTLSecondsAfterFinishing
+	if ttl == nil {
+		// do nothing if the cleanup delay is not set
+		return nil
+	}
+	duration := time.Second * time.Duration(*ttl)
```

Comment:
Please use loggerForTFJob
https://github.com/kubeflow/tf-operator/blob/master/pkg/controller.v2/controller_logger.go#L33

To construct a logger and then use that to log all messages. This way log messages will include metadata information such as the job it refers to.

Reply:
done